### PR TITLE
OpenSSH: No patch CVE-2007-2768

### PR DESCRIPTION
**Pull request by niontive**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `./.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

No patch CVE-2007-2768. Community agreed to not fix this.

See this discussion: https://bugzilla.suse.com/show_bug.cgi?id=628772&_ga=2.49772064.749617865.1604006580-702856597.1603302979

###### Change Log  

- No patch CVE-2007-2768

###### Does this affect the toolchain?  

No

###### Associated issues  

None

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2007-2768

###### Test Methodology

- Local build

### CODE REVIEW

File: SPECS/openssh/openssh.spec [hunk 1]
```diff
@@ -92,32 +98,32 @@ install -m644 contrib/ssh-copy-id.1 %{buildroot}/%{_mandir}/man1/
...
 %systemd_preun sshd.service sshd-keygen.service
 
 %post server
 /sbin/ldconfig
 if [ $1 -eq 1 ] ; then
-    chown -v root:sys /var/lib/sshd
+    chown -v root:sys %{_sharedstatedir}/sshd
 fi
-%systemd_post sshd.service sshd-keygen.service
+%{systemd_post} sshd.service sshd-keygen.service
```
Comment (thcrain-msft) [inline SPECS/openssh/openssh.spec:122]: The linter introduced a bug here- %systemd_post should not have brackets. This is updated in the latest version of the tool.

File: SPECS/openssh/openssh.spec [hunk 2]
```diff
@@ -92,32 +98,32 @@ install -m644 contrib/ssh-copy-id.1 %{buildroot}/%{_mandir}/man1/
...
 
 %postun server
 /sbin/ldconfig
-%systemd_postun_with_restart sshd.service sshd-keygen.service
+%{systemd_postun_with_restart} sshd.service sshd-keygen.service
```
Comment (thcrain-msft) [inline SPECS/openssh/openssh.spec:126]: The linter introduced a bug here- %systemd_postun_with_restart should not have brackets. This is updated in the latest version of the tool.

File: SPECS/openssh/openssh.spec [hunk 3]
```diff
@@ -127,16 +133,17 @@ if [ $1 -eq 0 ] ; then
     fi
 fi
 
-%clean
+%{clean}
```
Comment (thcrain-msft) [inline SPECS/openssh/openssh.spec:136]: The linter introduced a bug here- %clean should not have brackets. This is updated in the latest version of the tool.

**Review by thcrain-msft — CHANGES_REQUESTED** [review]
LGTM once the three comments are addressed.

**Comment by dmcilvaney** [comment]
Mentioned this in another PR, thoughts on splitting the linter fixes out from the CVE fixes into two commits, then doing a normal merge rather than a squash merge?

**Review by dmcilvaney — APPROVED** [review]

**Comment by dmcilvaney** [comment]
Fyi, conflict: https://github.com/microsoft/CBL-Mariner/pull/314/

**Comment by dmcilvaney** [comment]
Also, could you add `openssh` to the PR name and somewhere in the description? It makes it much easier to search github.

**Comment by niontive** [comment]
> Mentioned this in another PR, thoughts on splitting the linter fixes out from the CVE fixes into two commits, then doing a normal merge rather than a squash merge?

@PawelWMS Are you ok with me (a) adding this no-patch to your PR, and (b) closing this PR?

**Comment by PawelWMS** [comment]
> Mentioned this in another PR, thoughts on splitting the linter fixes out from the CVE fixes into two commits, then doing a normal merge rather than a squash merge?

Will the "normal" merge's commits still contain the information that they both are coming from this PR? If yes - go for it. If not, I'd personally vote for having one merge.

**Review by thcrain-msft — APPROVED** [review]

**Pull request merged**